python/openassetio/test/specifications.py

```diff
@@ -0,0 +1,31 @@
+#
+#   Copyright 2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+@namespace openassetio.test.specifications
+Specifications for use within openassetio test harnesses.
+"""
+
+from openassetio.specifications import LocaleSpecification
+
+class TestHarnessLocale(LocaleSpecification):
```

Comment:
[consider] PEP8 - 2 spaces above class :grin: 
```suggestion

class TestHarnessLocale(LocaleSpecification):
```